notification/notification-sdk/src/main/java/org/sunbird/notification/email/Email.java

```diff
@@ -194,11 +194,12 @@ public void sendAttachment(
    * @param bccList recipient bcc list
    * @return boolean
    */
-  public boolean sendEmail(String fromEmail, String subject, String body, List<String> bccList) {
+  public boolean sendEmail(String fromEmail, String subject, String body,  List<String> toList, List<String> bccList) {
     boolean sentStatus = true;
     try {
       Session session = getSession();
       MimeMessage message = new MimeMessage(session);
+      addRecipient(message, RecipientType.TO, toList);
```

Comment:
Why this is added as "TO" -- suppose to be "CC" ?

Reply:
This code has been modified and a new approach has been implemented.